src/templates/base/2019/base.html

```diff
@@ -323,81 +356,131 @@
-                {{ y }}
-              </option>
-              {% else %}
-              <option value="{{ url_for(request.endpoint, **get_view_args(lang=lang,year=y)) }}">
-                {{ y }}
-              </option>
-              {% endif %}
-            {% endfor %}
-            {# If the current page doesn't support a year, then show a home link for that year #}
-            {% if supported_years|length < all_supported_years|length %}
-              <option disabled="disabled" aria-hidden="true">
-                ────
-              </option>
-              {% for y in all_supported_years | sort %}
-              {% if y not in supported_years %}
-              <option value="{{ url_for('home', lang=lang,year=y) }}">
-                {{ y }} {{ self.home() }}
-              </option>
-              {% endif %}
-              {% endfor %}
-            {% endif %}
-        </select>
-      </div>
+      {% if switcher_name == "mobile" %}
+          {{ native_year_dropdown(switcher_name) }}
+        {% else %}
+          {{ styled_year_dropdown(switcher_name) }}
+      {% endif %}
     {% endif %}
   {% endmacro %}
 {% endif %}
 
+{% macro native_year_dropdown(switcher_name)%}
+  <div class="year-switcher">
+    <label for="year-switcher-{{switcher_name}}" class="visually-hidden">{{ self.year_switcher() }}</label>
+    <select id="year-switcher-{{switcher_name}}">
+      {% for y in supported_years | sort %}
+        {% if y == year %}
+          <option selected="selected" value="{{ url_for(request.endpoint, **get_view_args(lang=lang,year=y)) }}">
+            {{ y }}
+          </option>
+        {% else %}
+          <option value="{{ url_for(request.endpoint, **get_view_args(lang=lang,year=y)) }}">
+            {{ y }}
+          </option>
+        {% endif %}
+      {% endfor %}
+      {# If the current page doesn't support a year, then show a home link for that year #}
+      {% if supported_years|length < all_supported_years|length %}
+        <option disabled="disabled" aria-hidden="true">
+          ────
+        </option>
+        {% for y in all_supported_years | sort %}
+          {% if y not in supported_years %}
+            <option value="{{ url_for('home', lang=lang,year=y) }}">
+              {{ y }} {{ self.home() }}
+            </option>
+          {% endif %}
+        {% endfor %}
+      {% endif %}
+    </select>
+  </div>
+{% endmacro %}
+
+{% macro styled_year_dropdown(switcher_name)%}
+  <div class="nav-dropdown {{switcher_name}}">
+    <button class="nav-dropdown-btn" aria-expanded="false" aria-label="{{ self.year_switcher() }}">{{ year }}</button>
+    <ul class="nav-dropdown-list hidden">
+      {% for y in supported_years | sort %}
+        {% if y != year %}
+          <li>
+            <a href="{{ url_for(request.endpoint, **get_view_args(lang=lang,year=y)) }}">{{y}}</a>
+          </li>
+        {% endif %}
+      {% endfor %}
```

Comment:
We've lost our Home Page link  when chapter is not supported in future year.

![Chapter with 2019 but no 2020 version](https://user-images.githubusercontent.com/10931297/90344183-133b3500-e00f-11ea-833c-d0cb8d22c861.png)


```suggestion
      {% endfor %}
      <li>
        <a class="year-home-link" href="{{ url_for('home', lang=lang,year=y) }}"> {{ y }} {{ self.home() }}</a>
      </li>
```

And then also need ... separator style.

Reply:
This looks to be fixed now - thanks!